index.html

```diff
@@ -927,6 +927,9 @@
                 <li>If the pointer event dispatched was <code>pointerup</code> or <code>pointercancel</code>, clear the <i>PREVENT MOUSE EVENT</i> flag for this <code>pointerType</code>.</li>
             </ol>
             <div class="note">
+                <p>If the user agent supports both Touch Events (as defined in [[TOUCH-EVENTS]]) and Pointer Events, generation of compatibility mouse events as described in this section is not very web compatible. In this scenario gesture based generation of compatiblity mouse events for primary PointerEvents is better. For more information see [[TOUCH-EVENTS]].</p>
```

Comment:
The problem is that the gesture is not defined anywhere and I believe it is out of the scope of this spec to define it either. So I believe we agreed on a handwavy definition of sending events based on gestures and just a link to the touch event spec is okay. I used the sentence you suggested and just got rid of the rest of it. That first sentence seem to capture the essence of the note.